.github/workflows-source/ci-main.yml

```diff
@@ -115,7 +115,9 @@ jobs:
 
           # Determine which tests to skip and append 'long_test' for pull requests or merge groups
           EXCLUDED_TEST_TAGS=(system_test_hourly system_test_nightly system_test_nightly_nns system_test_staging system_test_hotfix system_test_benchmark fuzz_test fi_tests_nightly nns_tests_nightly)
-          [[ "${{ github.event_name }}" =~ ^(pull_request|merge_group)$ ]] && EXCLUDED_TEST_TAGS+=(long_test)
+          if [[ "${{ github.event_name }}" =~ ^(pull_request|merge_group)$ ]] && [[ "${RUN_ON_DIFF_ONLY:-}" == "true" ]]; then
```

Comment:
This will cause long_tests to run on the merge group since `RUN_ON_DIFF_ONLY` will be false in that case.

Reply:
So you probably want to check explicitly for:
`!contains(github.event.pull_request.labels.*.name, 'CI_ALL_BAZEL_TARGETS')`